public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -25,8 +25,9 @@ import {
 } from '@elastic/eui';
 
 type ContentPanelProps = {
-  title: string;
+  title: string | React.ReactNode | React.ReactNode[];
```

Comment:
Why we allow `React.ReactNode[]`?

Reply:
For the live chart, I do pass 2 nodes (EuiTitle + EuiBadge) to `title` parameter for `Live Anomalies` title and + `Live` Badge.  I guess we implemented `Live Chart` in different way. I think allowing `React.ReactNode[]` could add more flexibility, support current use case, at least no harm to the component.

Reply:
I copied this code snippet from Yizhe's working soln since I needed a change in the ContentPanel as well for my title and wanted to try to be consistent. @yizheliu-amazon let me know if you made any major changes to this

Reply:
I haven't made any major changes yet.

Reply:
Ok, I will leave this as is.